docs/guide/tournament-logistics.rst

```diff
@@ -318,16 +318,16 @@ If at all feasible you want to train that volunteers acting as runners and/or da
 
   - If runners are unfamiliar with debating in general, outline the basics of what draws are, what ballots are actually for, and what this process looks like from a debater's perspective.
   - Outline how/when the printing process occurs and who will sort/assign the ballots. Now is a good time to assign different runners to the different groups/rooms that they will be working with.
-  - It is critical that, as a group, you actually go to everyone one of the venue groups and identify all of the venue rooms that are listed so that everyone knows exactly where to go. This may take some time. But it is a good chance to both check those rooms actually exist and pre-identify any problems that might occur with runners and debaters finding them.
-  - Outline in general what happens during ballot collecting: when to do it, how to approach chairs, what do to if they are slow or delaying. You should raise the chance of chairs being belligerent and outline how they (and you) should deal with this.
-  - If you are having runners pre-check ballots it's a good idea to fill out a few 'bad' ballots to demonstrate the kinds of checking required. If you are using any communication systems (i.e. having runners mark of buildings as 'done' in an online system) go through that now also.
+  - It is critical that, as a group, you actually go to every one of the venue groups and identify all of the venue rooms that are listed so that everyone knows exactly where to go. This may take some time. But it is a good chance to both check those rooms actually exist and pre-identify any problems that might occur with runners and debaters finding them.
```

Comment:
Two options:
* "[...] you actually go to everyone of the venue [...]"
* "[...] you actually go to every venue [...]"

Splitting "everyone" isn't natural.

Reply:
"every one of", as a synonym for "each of", is idiomatic ([Grammarly](https://www.grammarly.com/blog/everyone-vs-every-one/), [Writing Explained](https://writingexplained.org/everyone-vs-every-one-difference), [Lexico/Oxford](https://www.lexico.com/definition/every_one), [Collins](https://www.collinsdictionary.com/dictionary/english/every-one))